src/jit/lclvars.cpp

```diff
@@ -1600,7 +1616,9 @@ void   Compiler::lvaCanPromoteStructVar(unsigned lclNum, lvaStructPromotionInfo
     
 #endif
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef can be removed since lvIsHfaRegArg direct reference is replaced with lvIsHfaRegArg() and that will return false for non-hfa targets.